README.md

```diff
@@ -115,14 +127,41 @@ For requesting the actor to the server you have to start the tracker. This way a
 TrackerAsset.Instance.Start();
 ```
 
+### Tracker Login and Start (asynchronous)
+
+To use the tracker asynchronously, you should use the methods LoginAsync and StartAsync.
+
+#### Tracker LoginAsync
+
+Tracker LoginAsync is not really necesary. If you are a developer, and you're logged into the Analytics server as a developer, you're going to receive traces either the player has logged or not. If you're a teacher, and you want to see traces in your activities, you can configure the activity to allow anonymous users.
+
+If you are a teacher and you want to use logged students, you have to add the students to the class and then ask them for credentials into your game and log them into the system using `TrackerAsset.Instance.LoginAsync(String username, String password)` function.
+
+```c#
+//Log in the student BEFORE starting the tracker
+String username = "student", password = "123456";
+
+TrackerAsset.Instance.LoginAsync(username, password);
+```
+
+#### Tracker StartAsync
+
+For requesting the actor to the server you have to start the tracker. This way actor is appended to the traces when generated. For starting the tracker you just have to call the `TrackerAsset.Instance.StartAsync()` function.
+
+```c#
+//Start the tracker before sending traces.
+
+TrackerAsset.Instance.StartAsync();
```

Comment:
The StartAsync method works similar to the Login and also has a callback, but in this case, no success is returned:

```csharp
TrackerAsset.Instance.StartAsync(() => {
    // Code to be executed after start
});
```